### PR TITLE
Fix debugger logic in r1.3

### PR DESCRIPTION
**Pull request by chihuahua**
In tensorflow/tensorflow#11952, I had set made some logic within debug_grpc_testlib return too early, breaking some debugger-related behavior. This PR fixes that.

### CODE REVIEW

**Comment by chihuahua** [comment]
@tensorflow-jenkins test this please

**Review by jart — APPROVED** [review]

**Comment by jart** [comment]
Jenkins test this please

**Comment by caisq** [comment]
The Linux GPU build is failing in an obscure way. Testing again

@tensorflow-jenkins test this please.

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
Ah - the GPU build failure can be ignored. It is due to an unrelated test tag change.

**Comment by caisq** [comment]
@chiahuahua, does this change need to be commit to the master branch as well?

**Comment by chihuahua** [comment]
@caisq, tensorflow/tensorflow#12012 already makes this change in master. Thanks for starting tests. Let me try again.

@tensorflow-jenkins test this please.
